lib/libspl/include/devid.h

```diff
@@ -29,80 +29,4 @@
 
 #include <stdlib.h>
```

Comment:
This stdlib.h include can be dropped as well.  I'm happy to see this all go!

We could consider removing the entire header file it's only included by `lib/libzfs/libzfs_import.c`, `lib/libzfs/libzfs_pool.c`, and `cmd/zpool/zpool_vdev.c`.  Well and `devname2devid.c` but more on that latter.  But let's leave it empty for now as you've done.